Formula/f/foxglove-cli.rb

```diff
@@ -0,0 +1,23 @@
+class FoxgloveCli < Formula
+  desc "Foxglove command-line tool"
+  homepage "https://github.com/foxglove/foxglove-cli"
+  url "https://github.com/foxglove/foxglove-cli/archive/refs/tags/v1.0.23.tar.gz"
+  sha256 "d03e708033cf7665ddec02625fc97380dbbb06177807f3c8b4d27f6a696bb348"
+  license "MIT"
+  head "https://github.com/foxglove/foxglove-cli.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    cd "foxglove" do
+      system "make", "build", "VERSION=v#{version}"
+      bin.install "foxglove"
+    end
+  end
+
+  test do
+    system "#{bin}/foxglove", "auth", "configure-api-key", "--api-key", "foobar"
```

Comment:
not quite sure why audit did not catch this

```suggestion
    system bin/"foxglove", "auth", "configure-api-key", "--api-key", "foobar"
```